app/services/fdc_service.rb

```diff
@@ -0,0 +1,7 @@
+class FDCService
+  def self.search_foods(search)
+    url = "https://api.nal.usda.gov/fdc/v1/foods/search?query=sweet%20potatoes&pageSize=10000&api_key=#{ENV['FDC_API_KEY']}"
+    response = Faraday.get(url)
```

Comment:
We would rather see you set these query params in a block that you create on the `get` method. This makes the request more readable and maintainable.